vision/benchmarks/accuracy/bench_module.py

```diff
@@ -164,8 +164,42 @@ def accuracy_get_module_roll(self, region: np.ndarray) -> float:
         """
         return get_module_roll(region)
 
+    def accuracy_saveCircleImage(
+        self, filename, color_image: np.ndarray, depth_image: np.ndarray
+    ) -> None:
+        """
+        Creates a folder and calls on the saveCircleImage function to save images with circles.
+
+        Parameters
+        ----------
+        color_image: ndarray
+            The color image.
+        depth_image: ndarray
+            The depth image.
+        filename
+            The file name.
+
+        Returns
+        -------
+        None
+        """
+        OUTPUT_FILE = "Circle_Images"
+        try:
```

Comment:
Since this function is called every time bench_module_accuracy runs the algorithms on an image, having it throw an error if the directory already exists creates unnecessary overhead. You can instead do something like draw_centers does here at the top of bench_module_accuracy:

`if draw_centers and not os.path.isdir(DRAW_CENTERS_DIR):`
`        os.mkdir(DRAW_CENTERS_DIR)`